universal-login-contracts/contracts/ERC1077.sol

```diff
@@ -118,7 +118,7 @@ contract ERC1077 is KeyHolder, IERC1077, IERC721Receiver {
         /* solium-disable-next-line security/no-call-value */
         (success, _data) = to.call.value(value)(data);
         bytes32 messageHash = calculateMessageHash(address(this), to, value, data, nonce, gasPrice, gasToken, gasLimit, operationType);
-        emit ExecutedSigned(messageHash, lastNonce-1, success);
```

Comment:
Why would we like to increase nonce? Note that we increase the nonce in line 114, therefore `nonce-1` equals to actual nonce.

Reply:
I just substituted `lastNonce-1` with `nonce`, which happen to be the same value as per lines 112 and 114, for performance reasons: saving 1 SLOAD and 1 SUB opcodes

Reply:
That would, unfortunately, change the semantics. 
We moved `lastNonce` incrementation before the call to avoid re-entry.

Reply:
@marekkirejczyk  this change doesn't remove the `lastNonce` increment, it's still there on the same line it always was.

All this change does is make use of the `nonce` argument already passed into the function instead of inlining an unnecessary `lastNonce-1` as `nonce` is already in scope and is equal to `lastNonce-1`

Reply:
Good point. My bad.